go.mod

```diff
@@ -24,12 +24,14 @@ require (
 	github.com/smallstep/cli v0.14.0-rc.3
 	github.com/smallstep/truststore v0.9.4
 	github.com/vulcand/oxy v1.0.0
-	github.com/yuin/goldmark v1.1.24
-	github.com/yuin/goldmark-highlighting v0.0.0-20200218065240-d1af22c1126f
+	github.com/yuin/goldmark v1.1.25
+	github.com/yuin/goldmark-highlighting v0.0.0-20200307114337-60d527fdb691
 	go.uber.org/zap v1.14.0
 	golang.org/x/crypto v0.0.0-20200302210943-78000ba7a073
 	golang.org/x/net v0.0.0-20200301022130-244492dfa37a
 	gopkg.in/natefinch/lumberjack.v2 v2.0.0
 	gopkg.in/square/go-jose.v2 v2.4.1 // indirect
 	gopkg.in/yaml.v2 v2.2.8
 )
+
+replace github.com/caddyserver/certmagic => ../../certmagic
```

Comment:
@mholt  Remove this -- should fix CI ;)

Reply:
D'oh. Not even sure how that got committed. Nice catch... sigh.

Reply:
FWIW I had to reset the FUZZIT_API_KEY too, so I'm not sure if I did that right. We'll see. I'm not even sure how to re-enable the checks, it seems I had to manually do a run on the pki branch from the Azure control panel.

Reply:
I think the Fuzzit API key wasn't affected because the job runs fine and shows progress

https://dev.azure.com/mholt-dev/Caddy/_build/results?buildId=964&view=logs&j=6b957288-3301-5435-9f73-758a5678d59e&t=a29c7330-30ca-5cb6-0170-5e46cdbd1b2e

Reply:
Ah good! Thanks for checking!